environment_setup.sh

```diff
@@ -442,15 +447,183 @@ if [ "$installed" == "" ] ; then
+    echo $'\n'
+
+    brew tap phinze/homebrew-cask
+    brew install brew-cask
+  fi
+
+  installed=`ls /Applications/ | grep -i Google\ Chrome`
+  if [ "$installed" == "" ] ; then
+    installed=`ls ~/Applications/ | grep -i Google\ Chrome`
+    if [ "$installed" == "" ] ; then
+
+      echo $'\n'
+      echo 'Installing Google Chrome'
+      echo $'\n'
+
+      brew cask install google-chrome
+    fi
+  fi
+
+    installed=`ls /Applications/ | grep -i Firefox`
+  if [ "$installed" == "" ] ; then
+    installed=`ls ~/Applications/ | grep -i Firefox`
+    if [ "$installed" == "" ] ; then
+
+      echo $'\n'
+      echo 'Installing Firefox'
+      echo $'\n'
+
+      brew cask install firefox
+    fi
+  fi
+
+  installed=`ls /Applications/ | grep -i Slack`
+  if [ "$installed" == "" ] ; then
+    installed=`ls ~/Applications/ | grep -i Slack`
+    if [ "$installed" == "" ] ; then
+
+      echo $'\n'
+      echo 'Installing Slack'
+      echo $'\n'
+
+      brew cask install slack
+    fi
+  fi
+
+  installed=`ls /Applications/ | grep -i iTerm`
+  if [ "$installed" == "" ] ; then
+    installed=`ls ~/Applications/ | grep -i iTerm`
+    if [ "$installed" == "" ] ; then
+
+      echo $'\n'
+      echo 'Installing iTerm2'
+      echo $'\n'
+
+      brew cask install iterm2
+    fi
+  fi
+
+  installed=`ls /Applications/ | grep -i 'Sequel Pro'`
+  if [ "$installed" == "" ] ; then
+    installed=`ls ~/Applications/ | grep -i 'Sequel Pro'`
+    if [ "$installed" == "" ] ; then
+
+      echo $'\n'
+      echo 'Installing Sequel Pro'
+      echo $'\n'
+
+      brew cask install sequel-pro
+    fi
+  fi
+
+  installed=`ls /Applications/ | grep -i PhpStorm`
+  if [ "$installed" == "" ] ; then
+    installed=`ls ~/Applications/ | grep -i PhpStorm`
+    if [ "$installed" == "" ] ; then
+
+      echo $'\n'
+      echo 'Installing PhpStorm'
+      echo $'\n'
+
+      brew cask install phpstorm
+    fi
+  fi
+
+  installed=`ls /Applications/ | grep -i 1Password`
+  if [ "$installed" == "" ] ; then
+    installed=`ls ~/Applications/ | grep -i 1Password`
+    if [ "$installed" == "" ] ; then
+
+      echo $'\n'
+      echo 'Installing 1Password'
+      echo $'\n'
+
+      brew cask install 1password
+    fi
+  fi
+
+  installed=`which zsh`
+  if [ "$installed" == "" ] ; then
+    echo $'\n'
+    echo "Installing ZSH"
+    echo $'\n'
+
+    brew install zsh
+  fi
+
+  installed=`ls ~/.oh-my-zsh | grep -i 'oh-my-zsh'`
+  if [ "$installed" == "" ] ; then
+    echo $'\n'
+    echo ' Installing Oh My ZSH'
+    echo $'\n'
+
+    curl -L https://raw.github.com/robbyrussell/oh-my-zsh/master/tools/install.sh | sh
+  fi
+
+  installed=`which hub`
+  if [ "$installed" == "" ] ; then
+    echo $'\n'
+    echo "Installing Hub"
+    echo $'\n'
+
+    brew install hub
+  fi
+
+  installed=`which terminus`
+  if [ "$installed" == "" ] ; then
+    echo $'\n'
+    echo "Installing Terminus"
+    echo $'\n'
+
+    composer require pantheon-systems/terminus
```

Comment:
Should be `composer global require` - and we need to get composer global bin into the path by putting this in zshrc/bashrc:

`export PATH=~/.composer/vendor/bin:$PATH`